pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
```

Comment:
AFAIK there isn't an init/setup command for this tool. Unless told otherwise, Ansible will simply read from a global config file ([`/etc/ansible/hosts`](http://docs.ansible.com/ansible/intro_inventory.html)).

In terms of simplicity, the following are good candidates:
- `command`: This is the bread and butter of this tool, making it possible to manage computers remotely
- `ping` or `setup`: These are simpler since it doesn't require any user input, but they're not the main reason someone would use Ansible

Note that `setup` does not actually perform any kind of initialization. It just [gathers facts about remote hosts](http://docs.ansible.com/ansible/setup_module.html)